ui/src/routes/devices.$id.settings.keyboard.tsx

```diff
@@ -51,7 +51,6 @@ export default function SettingsKeyboardRoute() {
       />
 
       <div className="space-y-4">
-        { /* this menu item could be renamed to plain "Keyboard layout" in the future, when also the virtual keyboard layout mappings are being implemented */ }
         <SettingsItem
           title="Paste text"
```

Comment:
Oh... I see, the comment refers to the `title="Paste text"` and blurb in the `<p>` below... when I get VK working with keyboard layouts, indeed we will need to come back here.

Reply:
While working on keyboard layouts for virtual keyboards, you might be introducing another setting _just_ for virtual keyboard layouts. But later this really should be merged into one common "Keyboard layout" setting, IMO.